llama-index-core/llama_index/core/workflow/context.py

```diff
@@ -11,6 +15,38 @@
     from .workflow import Workflow
 
 
+class ContextSerializer:
+    def __init__(self, allow_pickle: bool = False):
```

Comment:
I would do two different serializer classes, defaulting to the safer option not using pickle

Reply:
Done!